build_versions.json

```diff
@@ -6,6 +6,13 @@
       "db_version": "8.9.0",
       "r10k_version": "5.0.0",
       "rugged_version": "1.9.0"
+    },
+    {
+      "release": 7,
+      "server_version": "7.18.0",
+      "db_version": "7.21.0",
+      "r10k_version": "5.0.0",
```

Comment:
stupid question: which ruby do we use for r10k? the puppet agent ruby in 7 is 2.7. r10k 5 requires Ruby 3.1.

Reply:
havent tested this yet. its installen and configured with system ruby 3.2 in a build container. in main container ruby is not installed i think. and maybe i remove r10k completly and suggest the use of the r10k container

Reply:
it has not really something to do with this pr, so i created an issue for that and would like to handle this separately.

Reply:
dang it, it has... i copied the wrong part over from the other file m)